automation/cloud_logging.py

```diff
@@ -0,0 +1,79 @@
+"""Structured logging configuration that works in GCP.
```

Comment:
We don't have to do this, but it does make the logs much prettier (you get nicely formatted stacktraces, correlation info like ticket id, etc.). If something goes wrong in a polling function it's easier to tell what happened during that request in the past. I'd like to work it in to everything later, it's only on delivery email polling for now.